packages/wonder-blocks-data/util/request-tracking.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import * as React from "react";
+
+import type {IRequestHandler} from "./types.js";
+
+type TrackerFn = (handler: IRequestHandler<any, any>, options: any) => void;
+
+type HandlerCache = {
+    [type: string]: IRequestHandler<any, any>,
+    ...,
```

Comment:
TIL: This syntax is completely new to me. What's the purpose of it?

Reply:
This is marking this type as being explicitly inexact.  Flow is moving towards exact by default so they needed to introduce this syntax to allow for inexact objects in the future.